ocs_ci/framework/main.py

```diff
@@ -49,18 +49,25 @@ def init_ocsci_conf(arguments=None):
     parser.add_argument(
         '--ocs-version', action='store', choices=['4.2', '4.3']
     )
+    parser.add_argument('--ocs-registry-image')
     args, unknown = parser.parse_known_args(args=arguments)
-    if args.ocs_version:
-        version_config_file = os.path.join(
-            CONF_DIR, 'ocs_version', f'ocs-{args.ocs_version}.yaml'
-        )
-        args.ocsci_conf.insert(0, version_config_file)
+    ocs_version = args.ocs_version
     for config_file in args.ocsci_conf:
         with open(
             os.path.abspath(os.path.expanduser(config_file))
         ) as file_stream:
             custom_config_data = yaml.safe_load(file_stream)
             framework.config.update(custom_config_data)
+    ocs_registry_image = args.ocs_registry_image if (
```

Comment:
I might be nitpicking here but can we use standard multiline if here. Alternatively we could use `or` instead of the multiline if, i.e.
`ocs_registry_image = args.ocs_registry_image or framework.config.DEPLOYMENT.get('ocs_registry_image')`

Reply:
done

Reply:
I agree, while this might be idiomatic python, I would usually prefer something like this because it's more explicit and less error prone:

```
ocs_registry_image = framework.config.DEPLOYMENT.get('ocs_registry_image')
if args.ocs_registry_image:
    ocs_registry_image = args.ocs_registry_image
...
```

Anyway, this is a style/taste discussion (which I enjoy) rather than about correctness or functionality.